geoips/plugins/modules/procflows/single_source.py

```diff
@@ -1856,10 +1856,10 @@ def call(fnames, command_line_args=None):
 
     retval = 0
     if compare_path:
-        from geoips.geoips_utils import find_entry_point
+        from geoips.interfaces.module_based.output_checkers import output_checkers
 
-        compare_outputs = find_entry_point("output_comparisons", compare_outputs_module)
-        retval = compare_outputs(
+        output_checker = output_checkers.get_checker(final_products[0])
+        retval = output_checker.call(
```

Comment:
See comment above.  This could either be "output_checkers.find_checker" and return the actual plugin, as you are doing here.  Or separate it out into "output_checkers.identify_checker" and output_checkers.get(<checker_name>)

Reply:
Updated the function signature to be 'select_checker'